### ISSUE TITLE
Remove dependencies from external base non official image

### ISSUE THREAD

**Issue by diniz-victor** [arquivei/php-kafka-consumer#54]
Currently the project is using as a base image the https://hub.docker.com/r/joesantos386/laravel. It's bad because it creates an unwanted dependency and masks explicity about what is really used by the project. By removing it, it also will be possible to deprecate https://github.com/joesantos418/laravel-docker if you wish.

FYI: @joesantos418 @fcoedno

**Comment by joesantos418** [comment]
This image is used to run tests, @fcoedno said he intended on integrating the Dockerfile into this project and moving it from my dockerhub to arquivei's. I think it's a good idea.

**Comment by fcoedno** [comment]
Yeah, I've already started the implementation of this at #51 . I have already migrated the Dockerfile and added actions to build the base images, just need to take some time to understand the last pipeline errors.

The majority of the checks are already passing though

**Comment by diniz-victor** [comment]
Nice! I'll close this MR then.
